src/network/networkctl.c

```diff
@@ -2066,6 +2066,16 @@ static int link_status_one(
         if (r < 0)
                 return r;
 
+        r = sd_network_link_get_activation_policy(info->ifindex, &activation_policy);
+        if (r >= 0) {
+                r = table_add_many(table,
+                                   TABLE_EMPTY,
+                                   TABLE_STRING, "Activation Policy:",
+                                   TABLE_STRING,  activation_policy);
```

Comment:
Doubled white space after comma. Is it intended to 'align' the field with the above?

Reply:
wow, good eye! I don't think I intended it for alignment, I'm pretty sure it was just a typo :) will fix in the next push